lib-injection/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:latest
+
+WORKDIR /datadog-init
+
+ADD sitecustomize.py /datadog-init/sitecustomize.py
+ADD copy-lib.sh /datadog-init/copy-lib.sh
```

Comment:
We could remove `lib-injection/copy-lib.sh` and do this instead. Is this harder to read/maintain?

# Create `copy-lib.sh`
RUN touch /datadog-init/copy-lib.sh && echo '#!/bin/sh\n\ncp sitecustomize.py' >> /datadog-init/copy-lib.sh

Reply:
The copy-lib is the convention is used for all the libraries